setup.cfg

```diff
@@ -1,5 +1,5 @@
 [tool:pytest]
-addopts = --cov-report term-missing --cov=sphinx_gallery --durations=5 -rs
+addopts = --cov-report= --cov=sphinx_gallery --durations=5 -rs
```

Comment:
is the hanging `=` intentional? I'm not sure, just feels like it might be a big to me :-)

Reply:
Yes it's intentional, it generates the file but prints nothing by default. It makes debugging (much more common than coverage checking) much easier, and can still be easily overridden when someone wants the long report